### DISCUSSION TITLE
Do you need crunchyroll premium account to log in?

### DISCUSSION THREAD

**Discussion by Bariex** [jhassan8/crunchyroll-tizen#13] [Q&A] [answered]
Sorry, this is a stupid question
i’m planning to buy a membership on crunchyroll but i can’t log in to my account.

**Comment by jhassan8** [comment, marked as answer]
Hi, if you want to see the premium content, yes, but you can use a free account too... I just haven't added the premium tags to the content that you couldn't see yet... Let me know if you have any problems
